pkg/webui/console/containers/gateway-onboarding-form/gateway-provisioning-form/gateway-registration-form-section/validation-schema.js

```diff
@@ -26,7 +26,11 @@ const validationSchema = Yup.object().shape({
       .max(36, Yup.passValues(sharedMessages.validateTooLong))
       .matches(gatewayIdRegexp, Yup.passValues(sharedMessages.validateIdFormat))
       .required(sharedMessages.validateRequired),
-    eui: Yup.string().length(8 * 2, Yup.passValues(sharedMessages.validateLength)),
+    eui: Yup.string().test(
+      'length',
+      Yup.passValues(sharedMessages.validateLength)({ length: 16 }),
+      value => value && (value.length === 12 || value.length === 16),
```

Comment:
@kschiffer I removed both the debounce function and the state since the error message about the MAC conversion now appears instantly while input in focus, and we also need to display the button instantly.